src/components/Calendar/Calendar.tsx

```diff
@@ -210,6 +210,9 @@ const Calendar: React.FC<CalendarProps> = ({
 }) => {
     const [eventClick, setEventClick] = useState<CalendarEvent | null>(null);
     const calendarRef = useRef<FullCalendar>(null);
+    const [calendarStartTime] = useState("10:00:00");
+    const [calendarEndTime] = useState("15:00:00");
```

Comment:
If we're not changing these values then no need to have states for them. We can instead do
```
const calendarStartTime = "10:00:00";
```

Reply:
Updated